src/components/SectionTally.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import qs from 'query-string'
+import classNames from 'classnames'
+import SectionTallyCount from './SectionTallyCount'
+
+import styles from '../styles/SectionTally.css'
+import BarChart from './BarChart'
+import PieChart from './PieChart'
+import { generateChartDataFromSectionTally, CHART_TYPES } from '../util/sectionTally'
+
+const SectionTally = ({
+  source, campaign, autologin, rewardfulID,
+  tally, forceCollapse, showLabels,
+  small = false, horizontal = false, isBadge = false, showZero = true,
+  chartType = null, showLogo = true
+}) => {
```

Comment:
it might be nice if the user specifies the chartType such that it is non-empty but not in pie or bar than a little console.warning is thrown notifiiying the user of their misconfugration.